eve/main.yml

```diff
@@ -894,6 +894,7 @@ stages:
             - lifecycle-minor-version
             - lifecycle-major-version
             - bootstrap-restore
+            - single-node-install-rocky-8
```

Comment:
Hmm ok let's do it in post-merge for the moment, but I think we will likely want to move it in pre-merge in the future